odpf/optimus/runtime_service.proto

```diff
@@ -194,6 +194,11 @@ service RuntimeService {
             get: "/v1/project/{project_name}/job/{job_name}/replay/{id}"
         };
     }
+    rpc GetReplayList(GetReplayListRequest) returns (GetReplayListResponse) {
+        option (google.api.http) = {
+            get: "/v1/project/{project_name}/replay/list"
```

Comment:
we should make it follow REST principles, do we need `list`?

Reply:
updated to not use list